editor/input_event_configuration_dialog.cpp

```diff
@@ -140,6 +140,7 @@ void InputEventConfigurationDialog::_set_event(const Ref<InputEvent> &p_event, c
 								input_item->select(0);
 								input_list_tree->ensure_cursor_is_visible();
 								in_tree_update = false;
+								_input_list_item_selected();
```

Comment:
This will break Unicode mode action mapping if you enter the key first and then select a mode, since it will select a Latin key from the list and overwrite the Unicode value with it.